Source/DfBAdminToolkit/View/MainView.cs

```diff
@@ -201,6 +203,16 @@ public void UpdateTitleBarTeamStats()
             this.Refresh();
         }
 
+        public void CheckLatestVersion()
+        {
+            GitHubService service = new GitHubService();
+            Version currentVersion = System.Reflection.Assembly.GetExecutingAssembly().GetName().Version;
+            Version latestVersion = new Version(service.LatestVersion());
```

Comment:
@jackb-dropbox do you know if the RestSharp client makes the api call on a background thread? I don't want to block the main thread while we check for the latest version, (especially in the event that the user is not connected to the internet).  Maybe you can show me how to make it work that way.